storage/innobase/xtrabackup/test/t/check_for_passwords.sh

```diff
@@ -11,37 +11,26 @@ vlog "Preparing server"
 start_server
 mysql -e "CREATE USER pxb@localhost IDENTIFIED BY '$password_string'"
 mysql -e "GRANT ALL ON *.* TO pxb@localhost "
+mysql -e "CREATE TABLE t1(i int) engine=MYISAM" test
+mysql -e "INSERT INTO t1 values(1)" test
 mkdir -p $topdir/backup
 
 #pass option like password -p to be run with 
 function grep_in_datadir_ps() {
-	additional_options=$1
+  additional_options=$1
+  mysql -e "SELECT SLEEP(15) from t1 for update" test &
```

Comment:
I would make this a higher sleep, like 6000 and capture it's PID and kill it at the end of the backup. Jenkins can be unpredictable on this sense and 15 seconds might no be enough.

Reply:
added